xla/service/gpu/transforms/cudnn_custom_call_compiler.cc

```diff
@@ -276,6 +321,89 @@ absl::StatusOr<se::gpu::CudnnGraph> HloCustomCallToCuDnnGraph(
             config.fmha_scale(), dropout_rate > 0.0, bias != std::nullopt,
             dnn_mask_type, force_deterministic));
     return std::move(graph);
+  } else {
+    TF_ASSIGN_OR_RETURN(
+        auto gpu_config,
+        custom_call->backend_config<xla::gpu::GpuBackendConfig>());
+    xla::gpu::CudnnfMHABackendConfig &config =
+        *gpu_config.mutable_cudnn_fmha_backend_config();
+
+    int input_index = 0;
```

Comment:
Can you explain why you use `input_index++` instead of just hardcoding the value of the argument in each access?

I think it makes the code harder to read, since to understand what indexing is being done, we need to track the variable all the increments/decrements that happened since declaration.

Reply:
Removed input and output indexing as there are no variadic inputs or outputs.